resources/views/layouts/app.blade.php

```diff
@@ -12,23 +12,32 @@
 		<!-- Styles -->
 		<link href="{{ mix('css/app.css') }}" rel="stylesheet">
 
-	</head>
-	<body>
-		<div class="ui secondary pointing menu bluemarketheader" id="bluemarketheader">
-			<div class="right menu">
-				<a class="item {{ Request::is('/') ? 'active' : '' }}" href="{{ url('/') }}">
-					Home
-				</a>
-				<a class="item {{ Request::is('projects') ? 'active' : '' }}" href="{{ url('projects') }}">
-					Projects
-				</a>
-				<a class="item {{ Request::is('login') ? 'active' : '' }}" id="loginBtn" href="{{ url('login') }}">
-					Login
-				</a>
+        <title>Bluemarket - @yield('title')</title>
+    </head>
+    <body>
+        <div class="ui secondary pointing menu bluemarketheader" id="bluemarketheader">
+            <div class="right menu">
+                <a class="item {{ Request::is('/') ? 'active' : '' }}" href="{{ url('/') }}">
+                    Home
+                </a>
+                <a class="item {{ Request::is('projects') ? 'active' : '' }}" href="{{ url('projects') }}">
+                    Projects
+                </a>
+                <a class="item {{ Request::is('login') ? 'active' : '' }}" id="loginBtn" href="{{ url('login') }}">
+                    Login
+                </a>
+            </div>
+        </div>
+        <div class="container">
+			<div class="not-padded content">
+				@yield('not-padded content')
```

Comment:
¿Qué tanto esperamos tener not padded content? ¿Vale la pena agregarlo aquí? Por qué no mejor hacer esto vía CSS?